packages/universal-components/storybook/preview-head.html

```diff
@@ -1,11 +1,14 @@
-<link href="https://fonts.googleapis.com/css?family=Roboto:400,400i,500,500i,700" rel="stylesheet">
+<link
```

Comment:
Why is this file included?

Reply:
Because prettier kept insisting on changing it, even if I reverted the changes it was showing an error in vscode

Reply:
Are you using some kind of Prettier plugin? In our project it shouldn't care about `html` files because we don't use any parser for it.

Reply:
I am using Prettier - Code formatter.Apparently it was using its own settings for html but now disabled it for html files